pkgs/top-level/all-packages.nix

```diff
@@ -24489,6 +24489,8 @@ in
 
   magnetico = callPackage ../applications/networking/p2p/magnetico { };
 
+  mastodon-archive = python3Packages.callPackage ../applications/backup/mastodon-archive { };
```

Comment:
According to the readme, this is meant to also be used as an application. Do you mind adding the following to `pkgs/top-level/all-packages.nix`:
```
  <pkg-name> = with python3Packages; toPythonApplication <python-module>;
```
More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)

Reply:
Done.